index.html

```diff
@@ -1 +1,137 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3" crossorigin="anonymous">
+    <link rel="stylesheet" media="mediatype and|not|only (expressions)" href="print.css">
+    <title>FCC: Product landing</title>
+</head>
+<body>
+    <header class="header">
+        <div class="flex">
+            <div class="flex-1">
+                <img src="./images/—Pngtree—3d abstract piano keys and_6785737.png" alt="">
+            </div>
+            <div class="flex-5">
+                <a href="#" class="link-item">Features</a>
+                <a href="#" class="link-item">How It works</a>
+                <a href="#" class="link-item">Pricing</a>
+            </div>
+        </div>
+        <div class="title">
+            <h2 class="h-title">
+                Handcrafted, home-made masterpieces
+            </h2>
+            <input type="email" placeholder="Enter your email"><br>
```

Comment:
Kindly put your submit button and input in the same line 
Try as much as possible stick to Figma design to align  the button to their position